BraveWallet/Preview Content/MockStores.swift

```diff
@@ -11,30 +11,48 @@ import BraveShared
 
 extension WalletStore {
   static var previewStore: WalletStore {
-    .init(
+    let solTxManagerProxy = BraveWallet.TestSolanaTxManagerProxy()
+    solTxManagerProxy._makeSystemProgramTransferTxData = { _, _, _, completion in
+      completion(.init(), .success, "")
+    }
+    solTxManagerProxy._makeTokenProgramTransferTxData = {_, _, _, _, completion in
+      completion(.init(), .success, "")
+    }
+    
+    return .init(
       keyringService: MockKeyringService(),
       rpcService: MockJsonRpcService(),
       walletService: MockBraveWalletService(),
       assetRatioService: MockAssetRatioService(),
       swapService: MockSwapService(),
       blockchainRegistry: MockBlockchainRegistry(),
       txService: MockTxService(),
-      ethTxManagerProxy: MockEthTxManagerProxy()
+      ethTxManagerProxy: MockEthTxManagerProxy(),
+      solTxManagerProxy: solTxManagerProxy
     )
   }
 }
 
 extension CryptoStore {
   static var previewStore: CryptoStore {
-    .init(
+    let solTxManagerProxy = BraveWallet.TestSolanaTxManagerProxy()
+    solTxManagerProxy._makeSystemProgramTransferTxData = { _, _, _, completion in
+      completion(.init(), .success, "")
+    }
+    solTxManagerProxy._makeTokenProgramTransferTxData = {_, _, _, _, completion in
+      completion(.init(), .success, "")
+    }
```

Comment:
I ended up [making a `MockSolanaTxManagerProxy`](BraveWallet.TestSolanaTxManagerProxy) for SwiftUI previews. Either are fine I think, but maybe could make this one a `static var previewProxy: BraveWallet.TestSolanaTxManagerProxy` on `BraveWalletSolanaTxManagerProxy` so it can be re-used?